build.gradle

```diff
@@ -78,7 +78,7 @@ subprojects {
 
                 // Benchmark dependencies
                 jmh_generator_annprocess : "org.openjdk.jmh:jmh-generator-annprocess:$jmhVersion",
```

Comment:
Looks like this is causing all sorts of havoc because it's running the annotation processor and the bytecode generator. This annotation process should be removed in favor of just using the built-in dependency on the bytecode generator.

Note that I had to `rm -rf .gradle` after I removed this to get it to work smoothly.